.pipelines/nuget.yml

```
@@ -28,12 +33,24 @@ pr:
   branches:
     include:
       - "*"
-      
+
 stages:
-- template: templates/build-package-for-nuget.yml
-  parameters:
-    DoCompliance: ${{ parameters.DoCompliance }}
-    DoEsrp: ${{ parameters.DoEsrp }}
-    IsReleaseBuild: ${{ parameters.IsReleaseBuild }}
-    NugetVersionSuffix: ${{ parameters.NugetVersionSuffix }}
-    OrtNugetPackageId: 'Microsoft.ML.OnnxRuntime.Extensions'
+- ${{ if eq(parameters.IsDummyPackage, false) }}:
+  - template: templates/build-package-for-nuget.yml
+    parameters:
+      DoCompliance: ${{ parameters.DoCompliance }}
+      DoEsrp: ${{ parameters.DoEsrp }}
+      IsReleaseBuild: ${{ parameters.IsReleaseBuild }}
+      NugetVersionSuffix: ${{ parameters.NugetVersionSuffix }}
+
+- ${{ if eq(parameters.IsDummyPackage, true) }}:
```

nit: would `else` work?
```suggestion
- ${{ else }}:
```